src/Microsoft.TestPlatform.CoreUtilities/Output/OutputExtensions.cs

```diff
@@ -15,6 +15,13 @@ namespace Microsoft.VisualStudio.TestPlatform.Utilities
     public static class OutputExtensions
     {
         /// <summary>
+        /// Bool to decide whether Verbose level should be added as prefix or not in log messages.
+        /// </summary>
+        internal static bool AppendPrefix;
```

Comment:
I am not sure if an Extension methods unit should contain state or configuration.